Registry/registry.var

```diff
@@ -167,6 +167,12 @@ rconfig   logical   use_radar_rv            namelist,wrfvar4  1  .false.  - "use
 rconfig   logical   use_radar_rf            namelist,wrfvar4  1  .false.  - "use_radar_rf"             ""  ""
 rconfig   logical   use_radar_rqv           namelist,wrfvar4  1  .false.  - "use_radar_rqv"            ""  ""
 rconfig   logical   use_radar_rhv           namelist,wrfvar4  1  .false.  - "use_radar_rhv"            ""  ""
+rconfig   integer   radar_rf_opt            namelist,wrfvar4  1  1        - "radar_rf_opt"             ""  ""
```

Comment:
If these are really only for radar, perhaps these new fields should be in the `radar_da` namelist record.

Reply:
Each of these new namelist variables should have a quick description provided in `registry.var`.

Reply:
@davegill Historically, there were no good description for those radar namelist variables. I tried my best to provide some more info within PR message and will also add more info in WRF-V4.2's users guide for the users' benefit later time. But I will not add more description in registry.var file. Ideally, everything should be put into 'radar_da' record, but again it was historically done that way.  Will not try to improve this as I explained earlier in another message.

Reply:
Added some description for acronym rv, rf, rhv, rqv, ... etc.